src/components/WorldMap.js

```diff
@@ -7,12 +7,18 @@ import ReactMapGl, {
   NavigationControl,
 } from "react-map-gl";
 import "mapbox-gl/dist/mapbox-gl.css";
+import {
+  addMarkerToLocalStorage,
+  getMarkersFromLocalStorage,
+  removeMarkerFromLocalStorage,
+} from "../services/myMarkersStorage";
 
 export default function WorldMap() {
   const [countries, setCountries] = useState([]);
   const [visitedCountries, setVisitedCountries] = useState(0);
```

Comment:
maybe it's worth considering a name change here, as the value is a number, you could say numberOfVisitedCountries

when i read visitedCountries i expect an array with country names in it, even though i understand that's what's in the line above

Reply:
Yeah, it makes sense. Thanks Rikki